CMSgov/design-system
# [WNMGDS-865] Deprecate danger button

## Summary
- Deprecating the danger button variation from the documentation and adding warnings in the console and CSS when the danger button is used

### Deprecated
- Use of the danger button is deprecated and will show a warning in the console and in CSS however it will still work until fully deprecated
Demo URL
http://design-system-demo.s3-website-us-east-1.amazonaws.com/deprecate-danger-button 

## How to test
- View the button page at the demo URL and see that the danger button is no longer on the documentation site
- Add the `danger` variation prop or the `ds-c-button--danger` class to a button and see that it will still work but will show deprecated in the console and css if you use the inspector.
